src/components/general/ScheduleTile.vue

```diff
@@ -1,26 +1,224 @@
+            <p>{{ timestamp }}</p>
+          </div>
+          <div class="top-info-section">
+            <img src="@/assets/schedule/location.svg" alt="Location" />
+            <p>{{ location }}</p>
+          </div>
+          <div class="top-info-section">
+            <p v-for="tag in tags" :key="tag" class="tag">{{ tag }}</p>
+          </div>
+        </div>
+      </div>
+      <p>{{ description }}</p>
+      <div class="speakers">
+        <div v-for="(speaker,index) in speakers" :key="speaker">
+          <div class="image-container" @mouseleave="popUp = -1" @mouseover="open(index)">
+            <transition name="fade">
+              <div class="dialog" v-if="popUp===index">
+                <div class="name">{{speaker}}</div>
+                <div class="company">{{speakerInfo[speaker]["title"].split("@")[1]}}</div>
+                <div class="rectangle"></div>
+              </div>
+            </transition>
+            <img class="image" :src="getImage(speaker)" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script>
+// import Schedule from "@content/schedule.json";
+import Speakers from "@/content/speakers.json";
+import GoogleCalendar from "@/assets/schedule/GoogleCalendar.png";
+import OutlookCalendar from "@/assets/schedule/OutlookCalendar.png";
+import AppleCalendar from "@/assets/schedule/AppleCalendar.png";
 export default {
-	name: "ScheduleTile",
-	props: {
-		title: String,
-		description: String,
-		type: String,
-		location: String,
-		timestamp: String,
-		tags: Array,
-		speakers: Array
-	},
-}
+  name: "ScheduleTile",
+  props: {
+    title: String,
+    description: String,
+    type: {
+      default: "",
+      type: String
+    },
+    location: String,
+    timestamp: String,
+    tags: Array,
+    speakers: Array
+  },
+
+  data() {
+    return {
+      colors: {
+        "topic 1": "#8394F2",
+        "topic 2": "#F57A75",
+        "topic 3": "#17ADCE",
+        "topic 4": "#F9AFAB",
+        "": "#44AF69" // Misx
+      },
+      popUp: -1,
+      speakerInfo: Speakers,
+      showDropdown: false,
+      calendars: [
+        { name: "Google Calendar", icon: GoogleCalendar },
+        { name: "Outlook", icon: OutlookCalendar },
+        { name: "Apple Calendar", icon: AppleCalendar }
+      ]
+    };
+  },
+  methods: {
+    getImage(speaker) {
+      try {
+        return require("@/assets/speakers/" +
+          this.speakerInfo[speaker]["image"]);
+      } catch (e) {
+        // throw Error(`pic does not exist: ${pic}`);
+      }
+    },
+    open(popped) {
+      this.popUp = popped;
+    }
+  }
+};
 </script>
 
 <style scoped>
 .background {
-	background: grey;
+  margin: 1rem;
+  border: 1px solid grey;
+  border-radius: 16px;
+}
+
+.strip {
+  border-radius: 16px 0 0 16px;
+  position: absolute;
+  z-index: 5;
+  height: 32%;
+  margin-right: 1rem;
+  width: 1rem;
+  background-color: var(--type-color);
+}
```

Comment:
This strip is mispositioned on resize. Please position this without absolute positioning.
![image](https://user-images.githubusercontent.com/41529410/119604520-36c0b680-bdbd-11eb-8154-ec351b870594.png)